New log file opened at Thu Jan 07 21:02:46 IST 2016 by 2084@DESKTOP-U8DSLE2
Taking next connection to: 59442
New client attempting to connect on port: 59443-59442
  Address = /192.168.11.3:59442
This client on port 59443 has uuid 62e7d46d-0692-4415-b427-7753d0179c0d configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-62e7d46d-0692-4415-b427-7753d0179c0d
Taking next connection to: 59442
New client attempting to connect on port: 59542-59442
  Address = /192.168.11.3:59442
Handshake exception on socket: 59542
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 59442
New client attempting to connect on port: 59543-59442
  Address = /192.168.11.3:59442
This client on port 59543 has uuid 4ba631ca-582c-4d26-a911-59c3a0b5c958 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-62e7d46d-0692-4415-b427-7753d0179c0d, play-fork-run-4ba631ca-582c-4d26-a911-59c3a0b5c958
Taking next connection to: 59442
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 59442
